.github/workflows/pypi-test.yml

```
@@ -0,0 +1,60 @@
+name: test-pypi
+
+on:
+  push:
+    tags:
+      - '*.*.*'
+
+permissions:
+    contents: read
+
+jobs:
+  build:
+    name: Build distribution
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
+
+      - name: Install poetry
+        run: pip install poetry==1.8.4
```

```suggestion
      - name: Install poetry
        run: curl -sSL https://install.python-poetry.org | POETRY_VERSION=1.8.4 python3 -
```
to make sure Poetry is installed its own virtual environment.